lib/app/item.ex

```diff
@@ -2,25 +2,26 @@ defmodule App.Item do
   use Ecto.Schema
   import Ecto.Changeset
   import Ecto.Query
-  alias App.{Repo, Tag, ItemTag, Person}
+  alias App.{Repo, Tag, ItemList, ItemTag, Person}
```

Comment:
`ItemList`? -> `ListItem` 💭

Reply:
This schema represents the `join` table between `items` and `lists`. I can change the name of the table and schema :+1: